### ISSUE TITLE
Can You Change two Parameters in bitrate file? 

### ISSUE THREAD

**Issue by kenoshaYI** [irungentoo/Xiaomi_Yi_4k_Camera#58]
Just a quick question. For the 4k+ in the autoexec file I modifed the parameters to shoot 200 mps in the 4K 60P mode. Can I also change the parameters for 4K 30P mode in the same autoexec file. The 4K 30P mode is useful for time-lapses when I need them and I would like a faster bit rate. Thanks YIkenosha

**Comment by irungentoo** [comment]
yes, you can change the bitrate for all the modes in the same file and it will work.
